webapp/channels/src/actions/views/channel.ts

```diff
@@ -119,7 +122,7 @@ export function switchToChannel(channel: Channel & {userId?: string}): ActionFun
             getHistory().push(`${teamUrl}/messages/@${channel.name}`);
         } else if (channel.type === Constants.GM_CHANNEL) {
             const gmChannel = getChannel(state, channel.id);
-            getHistory().push(`${teamUrl}/channels/${gmChannel.name}`);
+            getHistory().push(`${teamUrl}/channels/${gmChannel?.name}`);
```

Comment:
You tell me, you approved the PR :stuck_out_tongue: https://github.com/mattermost/mattermost-webapp/pull/2078

Seems like old code, and hopefully can be removed. Will do some tests just in case.